gui/wxpython/gui_core/gselect.py

```diff
@@ -3044,7 +3031,7 @@ def _onClick(self, evt=None):
             if not self._chckMap():
                 self.buttonVecSelect.SetValue(False)
                 return
-        else:
+        else:  # noqa: PLR5501
```

Comment:
This could be reduced to _elif_ too, no?

Reply:
Reformulated the complete condition